### PR TITLE
Add first version of access statistics

### PR DESCRIPTION
**Pull request by Baschdl**
image

### CODE REVIEW

File: backend/apps/access_statistics/views.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import pandas as pd
+from datetime import datetime
+
+from match4healthcare.settings.common import RUN_DIR
+
+logged_data_names = ['time', 'status_line', 'status', 'request_time']
+threshold_to_filter = 10
+
+#@login_required
+#@staff_member_required
```
Comment (josauder) [inline backend/apps/access_statistics/views.py]: This should definitely be @login_required and @staff_member_required

**Review by josauder — DISMISSED** [review]
Protect the endpoint. Otherwise looks good to me

File: backend/apps/access_statistics/views.py (hunk 1 above)
Comment (bjrne) [reply backend/apps/access_statistics/views.py]:
```suggestion
@staff_member_required
```

File: backend/apps/access_statistics/views.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import os
+import pandas as pd
+from datetime import datetime
+
+from match4healthcare.settings.common import RUN_DIR
+
+logged_data_names = ['time', 'status_line', 'status', 'request_time']
+threshold_to_filter = 10
+
+#@login_required
```
Comment (bjrne) [inline backend/apps/access_statistics/views.py]:
```suggestion
@login_required
```

**Review by bjrne — APPROVED** [review]
Finally locally tested, works well. Keep in mind to lower the threshold for easy testing. :crocodile:

**Pull request merged**